packages/components/scrollbar/src/scrollbar.vue

```diff
@@ -151,6 +150,19 @@ export default defineComponent({
       { immediate: true }
     )
 
+    watch(
+      () => style.value,
+      () => {
+        if (!props.native)
+          nextTick(() => {
+            update()
+            if (wrap$.value) {
+              barRef.value?.handleScroll(wrap$.value)
+            }
```

Comment:
Is this `nextTick` necessary here? I think we should be a little bit careful with using `nextTick` in our code.

Reply:
> Is this `nextTick` necessary here? I think we should be a little bit careful with using `nextTick` in our code.

I need to make sure the dom update is complete, otherwise the height will be wrong